src/status_im/ui/screens/views.cljs

```diff
@@ -25,7 +25,7 @@
             [status-im.ui.screens.profile.contact.views :as profile.contact]
             [status-im.ui.screens.profile.group-chat.views :as profile.group-chat]
             [status-im.ui.screens.profile.photo-capture.views :refer [profile-photo-capture]]
-            [status-im.ui.screens.wallet.collectibles.views :as collectibles]
+            [status-im.ui.screens.wallet.collectibles.views :refer [collectibles-list]]
```

Comment:
why `:as` -> `:refer` ?

Reply:
just to be consistent with all other imports , we use `:refer` in this file, so , maybe we should refactor all this file then